### DISCUSSION TITLE
Retrieve intuit_tid from error response

### DISCUSSION THREAD

**Discussion by kborrow** [ruckus/quickbooks-ruby#575] [Q&A] [answered]
Is there a way to easily access the intuit_tid value when receiving an error via the API?

**Comment by ruckus** [comment, marked as answer]
Hi @kborrow you should be able to access the `intuit_tid` from the `Quickbooks::IntuitRequestException` instance

https://github.com/ruckus/quickbooks-ruby/blob/master/lib/quickbooks/service/base_service.rb#L424

Give that a shot.

**Reply by kborrow** [reply]
Thank you!  Just what I needed.
